Source/Classes/Core.js

```diff
@@ -130,15 +132,15 @@
 								Logger.info(data);
 							});**/
 							
-							if (os.platform() === 'win32') {
+							if (os2.platform() === 'win32') {
 								const ls = Process.spawn('D:\\Software\\Steam\\Steam.exe', [ '-applaunch', 322330, '-window' ], {
 									cwd: 'D:\\Software\\Steam\\',
 									detached: false
 								});
 							}
-							if (os.platform() === 'linux') {
-								const ls = Process.spawn(os.homedir() + '/.steam/steam/steamapps/common/Don''t Starve Together/bin/dontstarve', [ ], {
-									cwd: os.homedir() + '/.steam/steam/steamapps/common/Don''t Starve Together/bin/dontstarve',
+							if (os2.platform() === 'linux') {
+								const ls = Process.spawn('steam', [ 'steam://rungameid/322330'], {
```

Comment:
Don't see the need? I think I've already done that. The first commit was just testing to try and get things ready, although the 2nd commit was making sure things are right, unless I'm missing something I think you should be checking and edtting the final commit